activerecord/test/cases/arel/table_test.rb

```diff
@@ -8,12 +8,6 @@ class TableTest < Arel::Spec
       @relation = Table.new(:users)
     end
 
-    it "should create join nodes" do
```

Comment:
We can move this test to FactoryMethodTest, since the `create_string_join` method is defined in FactoryMethod